stable/external-dns/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: external-dns
-version: 2.10.3
+version: 2.10.4
```

Comment:
Since this repo follows [semver](https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#versioning) this looks like new functionality added - if so, please bump MINOR not PATCH.

Reply:
ok, i thought this should be taked as a bug. Just updated to 2.11.3. Thanks